boards/remote/include/board.h

```diff
@@ -25,36 +24,81 @@
 
 #include "cpu.h"
 #include "periph/gpio.h"
+#include "periph/spi.h"
 
 #ifdef __cplusplus
  extern "C" {
 #endif
 
 /**
- * @brief   LED pin definitions and handlers
+ * @name LED pin definitions
  * @{
  */
-#define LED0_PIN            GPIO_PIN(3, 2)
-#define LED1_PIN            GPIO_PIN(3, 5)
-#define LED2_PIN            GPIO_PIN(2, 3)
-
-#define LED0_MASK           (1 << 2)
-#define LED1_MASK           (1 << 5)
-#define LED2_MASK           (1 << 3)
-
-#define LED0_ON             (GPIO_D->DATA &= ~LED0_MASK)
-#define LED0_OFF            (GPIO_D->DATA |=  LED0_MASK)
-#define LED0_TOGGLE         (GPIO_D->DATA ^=  LED0_MASK)
-
-#define LED1_ON             (GPIO_D->DATA &= ~LED1_MASK)
-#define LED1_OFF            (GPIO_D->DATA |=  LED1_MASK)
-#define LED1_TOGGLE         (GPIO_D->DATA ^=  LED1_MASK)
-
-#define LED2_ON             (GPIO_C->DATA &= ~LED2_MASK)
-#define LED2_OFF            (GPIO_C->DATA |=  LED2_MASK)
-#define LED2_TOGGLE         (GPIO_C->DATA ^=  LED2_MASK)
```

Comment:
Well, until recently we went with `LED_RED_*` and `LED_GREEN_*` on all the boards even if they provide only blue LEDs or didn't provide any LED at all, to make it possible to call the LED macros from any application or library. However, this is somewhat confusing. By just enumerating the LED macros, this gets more consistent.